tests/apollo/test_skvbc_preexecution.py

```diff
@@ -370,6 +368,6 @@ async def test_alternate_f_isolated(self, bft_network, tracker):
     async def issue_tracked_ops_to_the_system(self, tracker):
         try:
             with trio.move_on_after(seconds=30):
-                await tracker.run_concurrent_ops(10, write_weight=1)
+                await tracker.run_concurrent_ops(50, write_weight=1)
```

Comment:
Instead of increasing the number of requests, let's add another parameter to the `@verify_linearizability` - `with_conflicts=False`. This parameter would then get propagated to the SkvbcTracker & PassThroughSkvbcTracker.
We could use it to modify the behavior of the `readset()` method - to return empty if `with_conflicts` is True.
PS: let's also drop the view change timeout parametrization from this PR

@yontyon what do you think?

Reply:
sounds great
On it

Reply:
done